code/datums/supplypacks.dm

```diff
@@ -1341,6 +1341,14 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 	cost = 15
 	containername = "bureaucracy crate"
 
+/datum/supply_packs/misc/tape
```

Comment:
_You think you hear a faint quacking from within..._

Reply:
🤔

Reply:
Make the tape crate have a random chance to contain a rubber ducky

Reply:
This... why didn't I think of this before

Reply:
Change committed 📯 

Also there's a bug right now where if you open / close a crate with a squeaky thing inside, the squeaky thing will squeak.

So if you open this you might get a nice quack to greet you 😄